lib/src/render.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:path/path.dart' as p;
+
+import 'ast/sass.dart';
+import 'exception.dart';
+import 'io.dart';
+import 'sync_package_resolver.dart';
+import 'visitor/perform.dart';
+import 'visitor/serialize.dart';
+
+/// Loads the Sass file at [path], converts it to CSS, and returns the result.
+///
+/// If [color] is `true`, this will use terminal colors in warnings.
+///
+/// If [packageResolver] is provided, it's used to resolve `package:` imports.
+/// Otherwise, they aren't supported. It takes a [SyncPackageResolver][] from
+/// the `package_resolver` package.
+///
+/// [SyncPackageResolver]: https://www.dartdocs.org/documentation/package_resolver/latest/package_resolver/SyncPackageResolver-class.html
+///
+/// Finally throws a [SassException] if conversion fails.
+String render(String path,
+    {bool color: false,
+    SyncPackageResolver packageResolver,
+    String indentType: 'space',
```

Comment:
For pure-Dart representations, let's make this `bool useSpaces` and keep the translation from JS-isms isolated in `lib/src/node.dart`.